src/core/stompest/util/__init__.py

```diff
@@ -22,6 +23,6 @@ def cloneFrame(frame, persistent=None):
     frame.unraw()
     headers = filterReservedHeaders(frame.headers)
     if persistent is not None:
-        headers[u'persistent'] = unicode(bool(persistent)).lower()
+        headers[u'persistent'] = unicode(bool(persistent)).lower() if version_info[0] == 2 else str(bool(persistent)).lower()
```

Comment:
Extract `cast = unicode if isPython3() else str`. Probably `cast`and `isPython2()` and `cast` should be `util` functions.